airbyte-config/models/src/main/java/io/airbyte/config/ConfigSchema.java

```diff
@@ -46,6 +46,12 @@
   STANDARD_SYNC_SUMMARY("StandardSyncSummary.yaml"),
   STANDARD_SYNC_SCHEDULE("StandardSyncSchedule.yaml"),
 
+  // worker
+  STANDARD_SYNC_INPUT("StandardSyncInput.yaml"),
+  NORMALIZATION_INPUT("NormalizationInput.yaml"),
+
+  STANDARD_SYNC_OUTPUT("StandardSyncOutput.yaml"),
```

Comment:
Should we just have an empty NormalizationOutput?

Reply:
i'm not sure it gets us anything to do so.